storey/drivers.py

```diff
@@ -549,6 +549,10 @@ def _build_simplified_feature_store_request(self, aggregation_element):
     @staticmethod
     def _convert_python_obj_to_expression_value(value):
         if isinstance(value, str):
+            # in order to handle ' in value.
+            # may impact performance
```

Comment:
please remove comment on performance (already discussed).
please add a comment that a string with both " and ' are not supported